app/views/clients/new.html.erb

```diff
@@ -442,17 +442,9 @@
             <label for="familyMembersRelation">Relation</label>
             <%= family_member_form.text_field :relation, placeholder: "", class: "form-control" %>
           </div>
-          <div class="col-md-3 mb-3"></div>
-        </div>
-        <div class="row">
-          <div class="col-md-3 mb-3"></div>
-          <div class="col-md-6 mb-3">
-            <label for="familyMembersDOB">Date of Birth</label>
-            <div class="card p-2">
-              <div class="input-group">
-                <%= family_member_form.date_select :dob, :start_year => 1901, :end_year => Time.current.year, class: "form-control" %>
-              </div>
-            </div>
+          <div class="col-md-1 mb-3">
+            <label for="familyMembersDOB">Age</label>
+              <%= family_member_form.text_field :dob, placeholder: "", class: "form-control" %>
```

Comment:
So a few comments on this change: This should probably be on its own branch and have its own pull request since it's not really related to the insurance drop down. This change will probably work, but on the database side of things, `familyMembersDOB` is being stored as a `varchar` where as if we make the change to `Age` it should be stored as an `int8`. So what you'll want to do, is make a rails migration that drops the `familyMembersDOB` column and adds a `familyMembersAge` column.